docs/notebooks/bald.pct.py

```diff
@@ -0,0 +1,144 @@
+# %%
+from dataclasses import astuple
+
+import gpflow
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+from gpflow.utilities import print_summary, set_trainable
+
+import trieste
+from trieste.acquisition.function import BayesianActiveLearningByDisagreement, PredictiveVariance
+from trieste.acquisition.rule import OBJECTIVE
+from trieste.models import create_model
+from trieste.models.gpflow.models import GaussianProcessRegression, VariationalGaussianProcess
+from trieste.objectives.utils import mk_observer
+from trieste.utils import map_values
+
+np.random.seed(1965)
+tf.random.set_seed(1965)
+
+sp = trieste.space.Box([-1, -1], [1, 1])
+inDim = 2
+
+
+def circle(x):
+    return tf.cast((tf.reduce_sum(tf.square(x), axis=1, keepdims=True) - 0.5) > 0, tf.float64)
+
+
+# %%
+from trieste.models.gpflow import GPflowModelConfig
+
+
+def create_bo_model(data):
+    # kernel = gpflow.kernels.SquaredExponential()
+    # kernel = gpflow.kernels.Matern52()
+    kernel = gpflow.kernels.SquaredExponential()
+    m = gpflow.models.VGP(astuple(data), likelihood=gpflow.likelihoods.Bernoulli(), kernel=kernel)
+    return trieste.models.create_model(
```

Comment:
lets use the VGP wrapper here directly

Reply:
using the VGP wrapper directly here seems to make the model quite hard to estimate the function. not sure if the problem is in the adam optimiser (I am still investigating this). Do you have a minimal working example for this wrapper for VPG with Bernoulli likelihood?